libs/sdk/src/common/push-protocol.ts

```diff
@@ -90,16 +109,16 @@ class PushProtocol {
     await this._notificationsStream.connect();
   }
 
-  async getSettingsNotification(): Promise<UserSetting[]> {
+  async getSettingsNotification(): Promise<unknown[]> {
     this.initialize();
     const subscriptions: Array<ChannelInfo> = await this._pushClient!.notification.subscriptions({
       channel: this._akashaChannelAccount,
     });
     if (!subscriptions.length) throw new Error('There is no settings for this channel');
     const channelSubscriptionInfo = subscriptions[0];
-    let userSettings: UserSetting[] = [];
+    let userSettings: unknown[] = [];
```

Comment:
This interface didn't exist in original branch so i used unknown to temporarily fix error